src/panels/lovelace/types.ts

```diff
@@ -104,11 +106,12 @@ export interface LovelaceTileFeature extends HTMLElement {
 
 export interface LovelaceTileFeatureConstructor
   extends Constructor<LovelaceTileFeature> {
-  getConfigElement?: () => LovelaceTileFeatureEditor;
   getStubConfig?: (
     hass: HomeAssistant,
     stateObj?: HassEntity
   ) => LovelaceTileFeatureConfig;
+  getConfigElement?: () => LovelaceTileFeatureEditor;
+  getConfigSchema?: () => HaFormSchema[];
```

Comment:
```suggestion
  getConfigSchema?: () => {schema: HaFormSchema[], computeLabel: (schema: HaFormSchema) => string};
```

Reply:
I would prefer automatic key generation like : `card.{card_name}.editor.fields.{field_name}.name` and let custom cards inject translations.

Reply:
How would injecting translations work? The card would register a function that we call with a language param?

Reply:
Yes, in `customCards` window object.